### ISSUE TITLE
There is no reaction to some of the options

### ISSUE THREAD

**Issue by dmkuzmin** [openscopeproject/InteractiveHtmlBom#457]
The following options do not work when the CLI script is called:
--highlight-pin1
--checkboxes
--bom-view
--layer-view

It doesn't matter what the value will be. After generating the ibom, these settings remain the same as they were configured in the last html document. It's like they're stored globally somewhere.

**Comment by qu1ck** [comment]
Yes, these settings are cached in browser. The flags affect the default state of the html page when you first open it. Try opening the file in an incognito window or use the reset config feature in html.

**Comment by dmkuzmin** [comment]
Yes, the reset works. Thanks
